src/uws_client.c

```diff
@@ -582,12 +586,14 @@ static void indicate_ws_open_complete_error_and_close(UWS_CLIENT_INSTANCE* uws_c
 static void indicate_ws_error(UWS_CLIENT_INSTANCE* uws_client, WS_ERROR error_code)
 {
     uws_client->uws_state = UWS_STATE_ERROR;
+    clear_pending_sends(uws_client);
```

Comment:
is it needed to clear pending frames? It was not in the original code. Checked the code, it seems that no sending and receiving will happen if the state is ERROR, so should be fine too.

Reply:
You're right, this is different behavior. I'll undo it, even though no interaction from the underlying IO should happen after this point.